schema.yaml

```diff
@@ -0,0 +1,236 @@
+                    enum:
+                      - monthly
+                      - weekly
+                  rrule:
+                    type: string
+                    pattern: '^((R|EX)RULE:[-=A-Z0-9;]*(\n|$))*'
+                required:
+                  - description
+                  - scheme
+                  - rrule
+              _source:
+                type: string
+            required:
+              - name
+              - city
+          events:
+            type: object
+            additionalProperties: false
+            patternProperties:
+              '^[0-9]{4}-[0-9]{2}-[0-9]{2}(-[-_a-zA-Z0-9]*)?$':
+                type: object
+                additionalProperties: false
+                properties:
+                  name:
+                    type: string
+                  start:
+                    oneOf:
+                      - format: date-time
+                      - format: date
+                  city:
+                    type: string
+                    pattern: '^[-a-z0-9]+$'
+                  venue:
+                    type: string
+                    pattern: '^[-a-z0-9]+$'
+                  venue-notes:
+                    type: string
+                    format: markdown
+                  description:
+                    type: string
+                    format: markdown
+                  talks:
+                    type: array
+                    items:
+                      type: object
+                      additionalProperties: false
+                      properties:
+                        title:
+                          type: string
+                        description:
+                          type: string
+                          format: markdown
+                        speakers:
+                          type: array
+                          items:
+                            type: string
+                        urls:
+                          type: array
+                          items:
+                            type: string
+                            format: uri
+                        coverage:
+                          type: array
+                          items:
+                            type: object
+                            additionalProperties: false
+                            minProperties: 1
+                            maxProperties: 1
+                            properties:
+                              slides:
+                                $ref: '#/definitions/uri'
+                              link:
+                                $ref: '#/definitions/uri'
+                              video:
+                                $ref: '#/definitions/uri'
+                              writeup:
+                                $ref: '#/definitions/uri'
+                              notes:
+                                $ref: '#/definitions/uri'
+                        lightning:
+                          const: true
```

Comment:
What exactly does this do? From the code, I'm unsure what's instance and what's value. Reading the [spec](https://epoberezkin.github.io/ajv/keywords.html#const) I again don't understand where's the keyword here.

Reply:
Oh, the "true" is the const, i.e. "lightning" can only be set to "true", never to "false"? Is that really necessary?

Reply:
Not necessary, but it's the one way to do it. To promote something to a full talk, remove the whole line.